src/lodash-moment.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Collection of all the utils in here. Add to this as you go.
+ */
+let lodashMoment = {};
+
+
+/**
+ * Check if a variable is a moment date object
+ *
+ * @method isMoment
+ * @param {*} value: Value to check
+ * @return {Boolean}
+ */
+export var isMoment = (value) => {
+  return (
+    _.isObject(value) &&
+    value._isAMomentObject === true
```

Comment:
Since we're now depending on lodash in here, maybe we should just alias `moment.isMoment` in case they're doing something more sophisticated.